src/mixins/filing-template-mixin.ts

```diff
@@ -66,11 +66,22 @@ export default class FilingTemplateMixin extends Vue {
    * @returns the IA Correction filing body to save
    */
   buildIaCorrectionFiling (isDraft: boolean): CorrectionFilingIF {
-    // if filing and paying, filter out removed orgs/persons and omit the 'action' property
+    // if filing and paying, filter out removed entities and omit the 'action' property
     let parties = this.getPeopleAndRoles
+    let shareStructure = this.getShareClasses
     if (!isDraft) {
       parties = parties.filter(x => x.action !== 'removed')
         .map((x) => { const { action, ...rest } = x; return rest })
+
+      // Filter out class actions
+      shareStructure = shareStructure.filter(x => x.action !== 'removed')
```

Comment:
Oh snap. All 3 of these should use the ActionTypes enum :-}

Reply:
And please add a comment for orgs/persons. Thx.

Reply:
oo right, i'll update them all 👍

Reply:
I wonder if lodash `omit()` instead of `map()` would improve this code?

Reply:
Hmm maybe? As far as speed/performance though i'm not sure which approach would be best. 
Updated to enum in next commit 👍

Reply:
Thanks for the enum update.

Since you repurposed the previous orgs/person code block, could you please add a comment to the parties code block similar to the other "Filter out..." ones? Thx.

Reply:
Definitely, will do!